draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -694,46 +694,56 @@ Hybrid KEM constructions aim to provide security by combining two or more
-ciphertext, even if the two candidate messages are chosen by the adversary
-himself.
-
-### Ciphertext Second Preimage Resistant (C2PRI) Security {#c2pri}
-
-The notion where, even if a KEM has broken IND-CCA security (either due to
-construction, implementation, or other), its internal structure, based on the
-Fujisaki-Okamoto transform, guarantees that it is impossible to find a second
-ciphertext that decapsulates to the same shared secret `K`: this notion is
-known as ciphertext second preimage resistance (C2SPI) for KEMs
-{{XWING}}.
-
-The same notion has also been described as chosen ciphertext
-resistance (CCR) {{CDM23}}.
+For KEMs, IND-CCA2 requires that no efficient adversary, given a ciphertext obtained
+by running Encaps
+with an honestly-generated public key, can distinguish whether it is given the
+"real" secret output from encaps, or a random string unrelated to the Encaps call
+that created that ciphertext. (Readers should note that this definition is slightly
+different than the corresponding one for public-key encryption {{RS92}}.)
 
 ## Binding Properties {#binding-properties}
 
-It is often useful for a KEM to have certain "binding properties", by which
-certain parameters determine certain others {{CDM23}}.  These properties are
-referred to with labels of the form X-BIND-P-Q.  For example, LEAK-BIND-K-PK
+It is often useful for a KEM to have certain "binding" properties, by which
+certain parameters determine certain others. Recent work {{CDM23}} gave
+a useful framework of definitions for these binding properties. Binding for KEMs
+is related to other properties for KEMs and public-key encryption, such as
+robustness {{GMP22}} TODO robust encryption paper cite {{ABNM}}, and collision-freeness TODO cite mohassel.
+
+The framework given by {{CDM23}} refers to these properties with labels
+of the form X-BIND-P-Q.  The first element X is the attack model---HON, LEAK, or MAL.
+P,Q means that given the value P, it is hard to produce another Q that causes Decaps to succeed.
+For example, LEAK-BIND-K-PK
 means that for a given shared secret (K), there is a unique encapsulation key
 (PK) that could have produced it, even if all of the secrets involved are
 given to the adversary after the encapsulation operation is completed (LEAK).
 
-The property LEAK-BIND-K,PK-CT is related to the C2PRI property discussed
-above.  Related to the ciphertext collision-freeness of the underlying PKE
-scheme of a FO-transform KEM. Also called ciphertext collision resistance.
+We treat LEAK-BIND-K-PK and LEAK-BIND-K-CT to be target binding properties.
+The HON-BIND security model seems too weak for real applications---real
+attacks in the LEAK model are known {{LEAK-ATTACK}}. We are not aware
+of any common settings where the MAL-BIND security model is needed; thus,
+LEAK-BIND seems a sensible middle ground.
+
+Implementors SHOULD think carefully about whether MAL-BIND attacks are
+feasible for their systems or protocols.
```

Comment:
SHOULD is normative language and since we haven't targeted MAL-BIND and aren't giving our reader explicitly MAL-BIND options in this document (some of the concrete instances in the other document may be) we should probably take this out